xwiki-batchimport-postprocessors/listidentifier/src/main/java/org/xwiki/batchimport/postprocessors/ListIdentifierPostprocessor.java

```diff
@@ -119,11 +120,26 @@ public void postProcessRow(Map<String, String> data, List<String> row, int rowIn
 
             }
         } catch (XWikiException e) {
-            logger.warn("Exception while post processing dblists for mapping " + mapping + " and config " + config
+            logger.warn("Exception while post processing lists for mapping " + mapping + " and config " + config
                     + " for row " + rowIndex, e);
         }
     }
 
+    protected String lookupKey(Map<String, String> keyByValueMap, String value, PropertyClass prop) {
+        String key = keyByValueMap.get(value);
+
+        if (StringUtils.isBlank(key) && prop instanceof StaticListClass) {
+            // For static lists, case insensitive trimmed value retrieval is attempted
+            // if no key is found for the value as is.
+            for (Map.Entry<String, String> entry : keyByValueMap.entrySet()) {
+                if (entry.getKey().compareToIgnoreCase(value.trim()) == 0) {
```

Comment:
I'd say the entry.getKey() should also be trimmed here, if a proper "normalized" comparison is the purpose, but it's not a big deal if it's not done.

Reply:
Indeed, I made sure to trim the key as well in commit 6223346.